CHANGELOG.md

```diff
@@ -2,6 +2,10 @@ Slurm Ground Motion Workflow
 # Changelog
 (Based on https://wiki.canterbury.ac.nz/download/attachments/58458136/CodeVersioning_v18p2.pdf?version=1&modificationDate=1519269238437&api=v2 )
 
+## [19.6.15] - 2019-11-01 -- Add timeshifting for LF
+### Added
+    - LF now has additional sim duration time added to account for time shifts to preserve causality 
```

Comment:
Bit more description would be useful. What was it before and what is it now and what versions of LF are supported.